ts/features/wallet/paypal/component/__test__/PayPalCheckoutPspComponent.test.tsx

```diff
@@ -23,11 +28,32 @@ describe("PayPalCheckoutPspComponent", () => {
     expect(component).toMatchSnapshot();
   });
 
+  it(`edit label should be always pressable`, () => {
+    const pspName = "pspName123";
+    const mockEditLabelOnPress = jest.fn();
+    const component = render(
+      <PayPalCheckoutPspComponent
+        onEditPress={mockEditLabelOnPress}
+        fee={fee}
+        pspName={pspName}
+        privacyUrl={"https://io.italia.it"}
+      />
+    );
+    expect(component).not.toBeNull();
+    const editLabel = component.queryByTestId("editLabelTestID");
+    if (editPspEnabled) {
```

Comment:
🚨 **issue**: if `editPspEnabled` is false, this entire test doesn't have a reason to exist. And it is [hardcoded `false`](https://github.com/pagopa/io-app/pull/3718/files#diff-0ab0b85d4953718579c65cd322891e9ad79df612548294ca0805277a8cd74ffcR38).

If we don't have the time to add an actual FF, I suggest commenting out the entire block and leave a TODO for when it will be actionable.

Reply:
It makes sense.
Done here ca6588cabd7323b4bcccdc37c593b4695562d0cd